src/java.base/share/classes/java/io/FileInputStream.java

```diff
@@ -85,7 +85,9 @@ public class FileInputStream extends InputStream
      * Creates a {@code FileInputStream} by
      * opening a connection to an actual file,
      * the file named by the path name {@code name}
-     * in the file system.  A new {@code FileDescriptor}
+     * in the file system.  {@linkplain java.nio.file##links Symbolic links}
+     * are automatically redirected to the <i>target</i> of the link.
+     * A new {@code FileDescriptor}
```

Comment:
In passing (and not for this PR) is that we should probably change the first sentence to say that it opens an existing file and drop the use of "connection".

Reply:
Tracked by [JDK-8340229](https://bugs.openjdk.org/browse/JDK-8340229).